src/components/KPIOptions.tsx

```diff
@@ -0,0 +1,148 @@
+import React from "react";
+import tw, { styled } from "twin.macro";
+
+import { useConnection, useCountdown, Time } from "../hooks";
+import { Info as UnstyledInfo } from "../assets/icons";
+import Heading from "./Heading";
+import Button from "./Button";
+import ProgressBar from "./ProgressBar";
+
+const DateCounter: React.FC<{ date: Time }> = ({ date }) => {
+  const { days, hours, minutes, seconds } = useCountdown(date);
```

Comment:
just a nit, because how you have this is perfectly fine, but i feel like this component could be stateless and just take props. would make this slightly more portable as its not dependent on useCountdown